discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
+            "examples": "yes",
+            "eventsAsync": "threaded",
```

Comment:
discord.py uses a mixture of threads and coroutines.

If the library internally uses threading for its async calls but it is not safe for a library user to call its API from a separate thread then it's threaded but not thread-safe. I can imagine some scenarios of this happening in e.g. C++ libs should they exist one day.

'thread-safe' implies that you can run the code in a thread to make it async. 'threaded' implies that the library already handles some aspects of it for you.